src/sbvr-api/odata-response.ts

```diff
@@ -158,9 +162,17 @@ export const process = async (
 	);
 
 	const odataIdField = sqlNameToODataName(table.idField);
+	const webResourceHandler = getWebresourceHandler();
 	for (const row of rows) {
 		for (const fieldName of processedFields) {
 			row[fieldName] = fetchProcessingFields[fieldName](row[fieldName]);
+			if (
+				webResourceHandler &&
```

Comment:
Potentially check the webResourceHandler before the loop and create a different function that is called in the loop.